src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -192,6 +198,9 @@ def create_subvolume(self, **kwargs):
                                 'quota': size
                             }
                             subvolume.set_attrs(subvolume.path, attrs)
+                            if kwargs['mode']:
```

Comment:
The `mode` should be updated to default mode (0755) on an idempotent operation if `mode` is not supplied. I know this is not followed for `uid` and `gid` but I think it should be done for uid/gid as well. Need to validate if it breaks existing applications (csi) using them before doing so for uid/gid.

Reply:
Updated code.
If mode is not specified then it will use default mode (0755)